src/OpenLoco/StationManager.cpp

```diff
@@ -80,12 +80,12 @@ namespace OpenLoco::StationManager
         company->var_8BB0[arg1] = byte_4F9462[arg1];
     }
 
-    static void sub_49E1F1(StationId_t id)
+    static void sub_49E1F1(StationId id)
     {
         auto w = WindowManager::find(WindowType::construction);
         if (w != nullptr && w->current_tab == 1)
         {
-            if ((addr<0x00522096, uint8_t>() & 8) && addr<0x01135F70, int32_t>() == id)
+            if ((addr<0x00522096, uint8_t>() & 8) && StationId(addr<0x01135F70, int32_t>()) == id) // _constructingStationId
```

Comment:
Is int32_t actually correct?

Reply:
Annoyingly it is. Its still being used in another location and I would be hesitant to change it until all locations are accounted for. Tbh there would be no difference in this exact situation though as its not assigning only reading.